.github/workflows/dev_pr/link.js

```diff
@@ -51,11 +68,40 @@ async function commentJIRAURL(github, context, pullRequestNumber, jiraID) {
   });
 }
 
+/**
+ * Adds a comment on the Pull Request linking the GitHub issue.
+ *
+ * @param {Object} github
+ * @param {Object} context
+ * @param {String} pullRequestNumber - String containing numeric id of PR
+ * @param {String} issueID - String containing numeric id of the github issue
+ */
+async function commentGitHubURL(github, context, pullRequestNumber, issueID) {
+  // Make the call to ensure issue exists before adding comment
+  const issueInfo = await helpers.getGitHubInfo(github, context, issueID, pullRequestNumber);
+  const message = "* Github Issue: #" + issueInfo.number
+  if (await haveComment(github, context, pullRequestNumber, message)) {
+    return;
+  }
+  if (issueInfo){
```

Comment:
I noticed that `commentJIRAURL` does not have the check for `issueInfo`. Is there a specific reason why we added it here? Should we add it to `commentJIRAURL` too?